hack/update-external-examples.sh

```diff
@@ -17,5 +39,41 @@ QUICKSTARTS_DIR="${OS_ROOT}/examples/quickstarts"
   # Specifically look for a line containing https://raw.githubusercontent.com, then
   # look for the first content in ()s on that line, which will be the actual url of the file,
   # then use curl to pull that file down.
-  curl -# $(grep https://raw.githubusercontent.com README.md | sed -E 's/.*\((.*)\) -.*/\1 -O/')
+  curl -# $(grep -E '\(https://raw.githubusercontent.com.*\)' README.md | sed -E 's/.*\((.*)\) -.*/\1 -O/')
+  rename -- '-example' '' *.json
+  
+  # rename templates that have a different name in openshift/library from what we had been naming
```

Comment:
Is there a point in the future where we can fix this?

Reply:
not that i foresee.  We can't change the actual metadata.name of the templates or we'd break people (because template.metadata.name is the name they use to reference the template when doing "oc new-app mytemplate" and such).  And I don't think we want to add logic in library to handle this (currently library just names the templates based on metadata.name which i think is the right thing to do).

this is why consistency is important :/